utils/platform/platform_interface.go

```diff
@@ -0,0 +1,100 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package platform
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/imdario/mergo"
+	"github.com/sealerio/sealer/common"
+	v1 "github.com/sealerio/sealer/types/api/v1"
+	v2 "github.com/sealerio/sealer/types/api/v2"
+	utilsnet "github.com/sealerio/sealer/utils/net"
+	"github.com/sealerio/sealer/utils/os/fs"
+	"github.com/sirupsen/logrus"
+)
+
+type Interface interface {
+	// Platform Get remote platform
+	Platform(host net.IP) (v1.Platform, error)
+}
+
+func GetClusterPlatform(cluster *v2.Cluster) (map[string]v1.Platform, error) {
+	clusterStatus := make(map[string]v1.Platform)
+	for _, ip := range cluster.GetAllIPList() {
+		IP := ip
+		ssh, err := GetHostSSHClient(IP, cluster)
+		//ssh, err := platform.GetHostSSHClient(IP, cluster)
+		if err != nil {
+			return nil, err
+		}
+		clusterStatus[IP.String()], err = ssh.Platform(IP)
+		if err != nil {
+			return nil, err
+		}
+	}
+	return clusterStatus, nil
+}
+
+func NewSSHClient(ssh *v1.SSH, isStdout bool) Interface {
```

Comment:
1. move new remote platform to remote_platform file, 
2. rename the func name with its truly does.